Samples/word-citation-management/src/commands/commands.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global global, Office, self, window */
+
+Office.onReady(() => {
+  // If needed, Office.js is ready to be called
+});
+
+/**
+ * Shows a notification when the add-in command is executed.
+ * @param event {Office.AddinCommands.Event}
+ */
+function action(event) {
```

Comment:
Should remove the commands folder entirely as it doesn't look like this code is ever called or used.